src/app/home/home-section1/home-section1.component.html

```diff
@@ -0,0 +1,19 @@
+<!-- this will be placed into a new component -->
+<section class="card">
+  <!-- bg image -->
+  <img src="assets/images/general/home-top-bg.jpg" class="card-img" alt="login-bg-image card-img">
+
+  <!-- content -->
+  <div class="card-body">
+    <!-- text -->
+    <h2 class="font700">Discover the <span class="clear-underline">world</span> and remember <span
+        class="clear-underline">every moment.</span></h2>
+    <!-- button -->
+    <ng-container *ngIf="!auth.isLoggedIn$.getValue()">
+      <button id="start-tracking-btn" class="btn btn-secondary" [routerLink]="['/user/login']">Start Tracking</button>
+    </ng-container>
+    <ng-container *ngIf="auth.isLoggedIn$.getValue()">
```

Comment:
I talk about getValue a little later in the review, but you should not reference a service that is pulled in from your component like this. Your component should call the auth service and set a property in your component for the view to read. something like:

```
this.isLoggedIn = this.auth.isLoggedIn$.pipe(take(1));
```

then your view would do:
```
<ng-container *ngIf="isLoggedIn | async">
```